storm/Error.cpp

```diff
@@ -3,6 +3,8 @@
 #include <cstdio>
 #include <cstdlib>
 
+static uint32_t s_lasterror = ERROR_SUCCESS;
```

Comment:
Since statics are allocated in an area of memory that's (as far as I know) always initialized to 0 on all modern systems, and since `ERROR_SUCCESS` is `0`, we can just leave this as the declaration `static uint32_t s_lasterror;` and drop the assignment.

Reply:
You are right, but I was thinking perhaps it would give some context as to what the default value should be?